HLTrigger/HLTfilters/plugins/L1TTkEleFilter.cc

```diff
@@ -147,8 +151,8 @@ bool L1TTkEleFilter::hltFilter(edm::Event& iEvent,
         passIsolation = true;
     }
 
-    if (offlinePt >= min_Pt_ && itkEle->eta() <= max_Eta_ && itkEle->eta() >= min_Eta_ && passQuality &&
-        passIsolation) {
+    if (offlinePt >= min_Pt_ && std::fabs(itkEle->eta()) < max_AbsEta1_ && std::fabs(itkEle->eta()) >= min_AbsEta1_ &&
```

Comment:
elsewhere in this file `std::abs` is used

https://github.com/cms-sw/cmssw/blob/a96c90c81f11a669e86a6da78e3df0f69a684bbb/HLTrigger/HLTfilters/plugins/L1TTkEleFilter.cc#L148-L149

I think that's the preferred option.